components/pdf_preview.jsx

```diff
@@ -158,10 +158,12 @@ export default class PDFPreview extends React.PureComponent {
         }
 
         if (this.state.pdf.numPages > MAX_PDF_PAGES) {
+            const fileDownloadUrl = this.props.fileInfo.link || getFileDownloadUrl(this.props.fileInfo.id);
```

Comment:
I gave priority(don't know if it's required) to `fileInfo.link` because I saw the same use in different places in the code base like this one: https://github.com/mattermost/mattermost-webapp/blob/52bcb81/components/view_image/view_image.jsx#L215